### ISSUE TITLE
Installation Error

### ISSUE THREAD

**Issue by ChinwalPrasad** [valet-lite/beta#19]
Device: MacBook Pro (Late 2016)
OS: MacOS Catalina 10.15.7

Hey David,

So I did a fresh install of Valet Lite and the installation went perfect. I was asked to install some dependencies by clicking a button. Once clicked I was asked to grant privileges which I did. It looked like there was some download going on when suddenly I received the error message below.

<img width="423" alt="Screen Shot 2020-10-11 at 11 43 17 AM" src="https://user-images.githubusercontent.com/40767861/95684458-88625b00-0bb7-11eb-8f02-4e60ca4ed075.png">

Let me know if you need more information from my end.

Best,
Prasad Chinwal

**Comment by davidhemphill** [comment]
Can you reproduce this? Valet Lite doesn't do a download, so not sure what would've happened here. Looks like something involving a menu. :/

**Comment by ChinwalPrasad** [comment]
Hey David, weirdly this time its a different issue. So I did a fresh install again with Valet-Lite-0.1.27.dmg file.
On being asked to install an update I clicked yes.

<img width="414" alt="Screen Shot 2020-10-11 at 3 31 00 PM" src="https://user-images.githubusercontent.com/40767861/95689597-801a1800-0bd7-11eb-8ef8-5c7dfb69551c.png">

And once I clicked yes I got the error below.
<img width="427" alt="Screen Shot 2020-10-11 at 3 31 26 PM" src="https://user-images.githubusercontent.com/40767861/95689639-b9528800-0bd7-11eb-8c23-e0be6735ffef.png">
